commerce_coordinator/apps/order_fulfillment/serializers.py

```diff
@@ -0,0 +1,17 @@
+"""Serializers for LMS (edx-platform) service"""
+from rest_framework import serializers
+
+from commerce_coordinator.apps.core.serializers import CoordinatorSerializer
+
+
+class FulfillOrderWebhookSerializer(CoordinatorSerializer):
+    """
+    Serializer for Fulfill Order Webhook input validation.
+    """
+    fulfillment_type = serializers.CharField(required=True)
+    entitlement_uuid = serializers.CharField(required=False)
```

Comment:
what if it was not entitlement fulfillment? this shouldn't be required, since it will only be available for entitlement fulfillments.